### PR TITLE
[profile] profile page styling changes

### PR DESCRIPTION
**Pull request by arfamomin**
### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

I've extended the logout button, actions container, header, title, and user info container to 84% width, removed use of margin left/right when used to for container size and placement, and fixed alignment issues. 

## How to review

[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

Edited files: styles.ts, index.tsx in profile
Reviewed all at once. Try testing on different sized phones/screens to make sure the styling is flexible (looked good when I did it on Folasade's phone). 

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # 'Optional - copy links to any tutorials or documentation that was useful to you when working on this PR'
N/A

### Related PRs
[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @ronniebeggs

### CODE REVIEW

File: src/app/(BottomTabNavigation)/Profile/styles.ts [hunk 1]
```diff
@@ -105,9 +105,10 @@ export default StyleSheet.create({
   },
   signOutButton: {
     alignSelf: 'flex-start',
-    width: 125,
+    width: 320,
```
Comment (ronniebeggs) [inline src/app/(BottomTabNavigation)/Profile/styles.ts]: I'd recommend starting by creating a new "content-containter" `View` directly nested under the outermost "container". Give this container a percentage width (most of our screens have a 84% width) such that it's centered relative to the screen's width. Having an outermost container like this will allow us to consistently set the width of all inner buttons/views without using pixel estimates.

File: src/app/(BottomTabNavigation)/Profile/styles.ts [hunk 2]
```diff
@@ -105,9 +105,10 @@ export default StyleSheet.create({
...
     height: 45,
-    marginLeft: 27,
+    marginLeft: 29,
+    marginRight: 27,
```
Comment (ronniebeggs) [inline src/app/(BottomTabNavigation)/Profile/styles.ts]: I'm trying to get our team trending away from using margin values, as they don't scale well on different screens. I'd recommend using flex properties to align the items within the `TouchableOpacity`. It may be helpful to put the "Log out" text and icon within a sub-View, and move it around within the box using the "justify-content" and "align-items" CSS properties.

**Review by ronniebeggs — CHANGES_REQUESTED** [review]
Looking good so far!!! I left a few comments and potential hints for moving forward with this PR. You don't have to restyle the entire screen if it feels like too much. I'm trying to indoctrinate you with good styling practices, so I apologize if it feels a little over the top lol. I think the best way to learn CSS is by seeing other people's work, and just experimenting. 

Also, add questions/comments to the comment thread below the PR, so you don't have to repeatedly update the PR template!

File: src/app/(BottomTabNavigation)/Profile/styles.ts (hunk 1 above)
Comment (arfamomin) [reply src/app/(BottomTabNavigation)/Profile/styles.ts]: Did this! Thanks for hint

File: src/app/(BottomTabNavigation)/Profile/styles.ts (hunk 2 above)
Comment (arfamomin) [reply src/app/(BottomTabNavigation)/Profile/styles.ts]: I went through and got rid of left/right margin for everything except what's contained inside the actions/userInfo container and replaced it with percentage widths. Working on adjusting the log out button now. Any ideas that won't require using margin/padding?

Comment (arfamomin) [reply src/app/(BottomTabNavigation)/Profile/styles.ts]: Also noticed the StyledButton Component --> Is there a possibility to created a modified component that matches the style for the logout button with the icon, text, and arrow? Or should I just work within the profile styling files

Comment (ronniebeggs) [reply src/app/(BottomTabNavigation)/Profile/styles.ts]: I'd say just work within the profile styling files. That way you gain some experience styling. We can standardize components and everything later.

Comment (ronniebeggs) [reply src/app/(BottomTabNavigation)/Profile/styles.ts]: In this case, I think "paddingHorizontal" will probably be fine, if it's around 20 px (you can check the spacing using figma dev mode). But within those bounds, try using the `justifyContent: "space-between"` option, which will push all elements within the `TouchableOpacity` as far away from each other as possible.

Comment (ronniebeggs) [reply src/app/(BottomTabNavigation)/Profile/styles.ts]: We also want to include the "right-caret.svg" icon in the logout button. Observe how the `SignOut` icon is imported and used and try to replicate it.

Screen Shot 2024-03-02 at 4 46 53 PM

Comment (ronniebeggs) [reply src/app/(BottomTabNavigation)/Profile/styles.ts]: One last thing, if we want the the "Log out" text and icon to appear together and on the left side, how could we "group" them?

**Comment by ronniebeggs** [comment]
See if you can apply similar styling/spacing techniques to the other parts of the screen. Good job so far!
